en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/configuration-overview.md

```diff
@@ -2,16 +2,76 @@
 
 The following are the Choreo Connect related configuration files.
 
-| **File name**            | **Description**                                                                                              | **Directory (File path)**                                              |
-|--------------------------|--------------------------------------------------------------------------------------------------------------|----------------------------------------------------------- |
-| `config.toml`            | This file defines all the adapter, enforcer, router, security, control plane and analytics configurations.   | `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf` |
-| `log_config.toml`        | This file defines the logging configurations for the Adapter and Router.                                              | `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf` |
-| `log4j2.properties`      | This file defines the logging configurations for the Enforcer.                                                    | `<CHOREO-CONNECT_HOME>/docker-compose/choreo-connect/conf` |
+| **File** | **File Name Example** |
```

Comment:
I feel like this column is more like a description
```suggestion
| **File** | **Description** |
```

Reply:
+1